src/coreclr/src/jit/compiler.cpp

```diff
@@ -4575,18 +4767,18 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
     }
 #endif // _TARGET_ARM_
 
-    /* Assign registers to variables, etc. */
+    // Assign registers to variables, etc.
 
     ///////////////////////////////////////////////////////////////////////////////
     // Dominator and reachability sets are no longer valid. They haven't been
     // maintained up to here, and shouldn't be used (unless recomputed).
     ///////////////////////////////////////////////////////////////////////////////
     fgDomsComputed = false;
 
-    /* Create LSRA before Lowering, this way Lowering can initialize the TreeNode Map */
+    // Create LSRA before Lowering, this way Lowering can initialize the TreeNode Map
```

Comment:
Nit: since you're here, the TreeNode map is gone; I would reword this as something like  "Create LinearScan before Lowering, so that Lowering can call LinearScan methods for determining whether locals are register candidates and (for xarch) whether a node is a containable memory op.

Reply:
Updated